blog/_posts/2020-12-12-multi-cluster-monitoring.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: post
+title: Multi-Cluster monitoring with Prometheus and inlets
```

Comment:
Multi-cluster monitoring with Prometheus and inlets -> Monitor multiple remote clusters with Prometheus and Grafana (second one is more use-case focused)